examples/tutorials/core_api_pytorch_mnist/model_def_checkpoints.py

```diff
@@ -166,7 +181,9 @@ def main(core_context):
     latest_checkpoint = info.latest_checkpoint
     if latest_checkpoint is not None:
         with core_context.checkpoint.restore_path(latest_checkpoint) as path:
-            model = load_state(path)
+            model, epochs_completed = load_state(path, info)
+    else:
+        epochs_completed = 0
```

Comment:
nit: slight preference for reversing the `if` block order:

```python
if latest_checkpoint is None:
  epochs_completed = 0
else:
  with core_context.checkpoint.restore_path(...):
    ...
```

Reason: it's easier to read code if the physical code layout mirrors the execution order.  The way I am proposing it, the first time you run the code you hit the first block, the second time (if you pause/activate) you'll go through the second block.